cmd/ncproxy/ncproxy.go

```diff
@@ -57,6 +59,86 @@ func (s *grpcService) AddNIC(ctx context.Context, req *ncproxygrpc.AddNICRequest
 	return nil, status.Errorf(codes.FailedPrecondition, "No shim registered for namespace `%s`", req.ContainerID)
 }
 
+func (s *grpcService) ModifyNIC(ctx context.Context, req *ncproxygrpc.ModifyNICRequest) (_ *ncproxygrpc.ModifyNICResponse, err error) {
+	ctx, span := trace.StartSpan(ctx, "ModifyNIC")
+	defer span.End()
+	defer func() { oc.SetSpanStatus(span, err) }()
+
+	span.AddAttributes(
+		trace.StringAttribute("containerID", req.ContainerID),
+		trace.StringAttribute("endpointName", req.EndpointName),
+		trace.StringAttribute("nicID", req.NicID))
+
+	log.G(ctx).WithField("iov settings", req.IovPolicySettings).Info("ModifyNIC iov settings")
```

Comment:
Do we need this anymore? Can we just have this in the span?

Reply:
This

Reply:
we can't add it to the span because it's not a string, int/float, or other basic type

Reply:
Just add the individual fields I meant

Reply:
Ok they're all int64s, just leave it actually